### ISSUE TITLE
Error when parsing 'volatile'

### ISSUE THREAD

**Issue by trelau** [robotpy/cxxheaderparser#16]
In this code:

```cpp
class Foo
{
public:

private:

volatile bool               myToShutDown;

}
```

Gives error:
```
cxxheaderparser.errors.CxxParseError: Not able to parse volatile_error.hxx on line 7 evaluating 'volatile'
```

**Comment by trelau** [comment]
does this PR address this issue? https://github.com/robotpy/cxxheaderparser/pull/8

btw, still tracking this project as I think it might be an easier way to generate bindings for https://github.com/trelau/pyOCCT (currently using clang)

**Comment by virtuald** [comment]
Yes, it addresses that issue. I'll add an explicit test too.

I'm still planning on migrating robotpy-build to use this, but I ran out of time last season. Expecting to pick up the work this fall, but no firm timeline.

As a sidenote, when I merged that fix, I was trying to utilize this for codegen and I found that cxxheaderparser wasn't very user-friendly. Will have to strategize on ways to address that.
